### ISSUE TITLE
Many LotV Replays Don't Work

### ISSUE THREAD

**Issue by Reithan** [Reithan/PACAnalyzer#4]
It seems development of Sc2Reader hasn't kept up and LotV is producing many replay parsing errors now. Additionally, we should have better error handling in these cases, including 'known' issues such as no actions recorded for AI players.

**Comment by Kaszanas** [comment]
@Reithan This is true. I have found original Sc2Reader to have had stopped its development. Although the creators of Sc2Reader simply forked it over into a different account and are still continuing the development.
https://github.com/ggtracker/sc2reader - this is the new repository that is keeping up with the newest updates.
I think they are 1 update behind the current protocol.
